docker-compose.yml

```diff
@@ -292,6 +292,12 @@ services:
     depends_on:
       elasticsearch:
         condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s -f http://kibana:5601/api/status >/dev/null || exit 1"]
```

Comment:
`http://localhost:5601/api/status` is IMO much better because it's using localhost instead of the service name